test/Microsoft.AspNet.Mvc.Core.Test/ApplicationModel/DefaultControllerModelBuilderTest.cs

```diff
@@ -220,4 +271,27 @@ public class NoSuffix : Mvc.Controller
     public class PocoController
     {
     }
+
+    [Produces("application/json")]
```

Comment:
Controller with a filter type that is not supported does not show up (and consider if we should log/trace)